tensorflow/compiler/jit/mark_for_compilation_pass.cc

```diff
@@ -45,6 +45,42 @@ const char* const kXlaOutsideCompilationAttr = "_XlaOutsideCompilation";
 
 namespace {
 
+// An node will not be processed by XLA even it has the XLA kernel:
+// 1, In Tensorflow, conventionally, INT32 ops are regarded as shape or control ops, and
```

Comment:
Can you say more about why you want to do this? Do you have an example where int32 copies cause a problem?

I am reluctant to do this because it is something of a historical accident that the TF classic GPU device doesn't have int32 kernels, and ideally it is something that we would fix. I think a more specific motivating example would help clarify things for me here.